checks/raw/github/packaging.go

```diff
@@ -73,7 +75,12 @@ func Packaging(c *checker.CheckRequest) (checker.PackagingData, error) {
 
 		runs, err := c.RepoClient.ListSuccessfulWorkflowRuns(filepath.Base(fp))
 		if err != nil {
-			return data, fmt.Errorf("Client.Actions.ListWorkflowRunsByFileName: %w", err)
+			// appending empty run for localdir client
```

Comment:
lets clarify the purpose of adding the empty run in the comment. maybe something like (but open to suggestions):

```go
// assume the workflow will run for localdir client
```